ready_recipes/arm_pyart/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.6.0" %}
+
+package:
+    name: arm_pyart
+    version: {{ version }}
+
+source:
+    fn: arm_pyart-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/a/arm_pyart/arm_pyart-{{ version }}.tar.gz
+    md5: 6c382dda78193c730a6a47d2d7b18eeb
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - trmm_rsl   # [not win]
+
+    run:
+        - python
+        - numpy
```

Comment:
The reason for the unsatisfiable will be one of:

```
+        - scipy
+        - matplotlib
+        - netcdf4
+        - trmm_rsl  # [not win]
```

My assumption is that one of those doesn't have an equivalent numpy 1.9 build. If you want to get to the bottom of it, consider putting the run requirements into the build requirements and seeing which items in the matrix don't get built (because there would be unsatisfiable dependencies, and conda-build-all deals with that).